src/models/score/score-data-mapper.ts

```diff
@@ -84,6 +86,20 @@ export class ScoreDataMapperImpl extends GenericDataMapper implements IScoreData
     ).toPromise();
   }
 
+  public async deleteScoring(uid: number, judge: string): Promise<IDbResult<void>> {
+    const query = squel.delete({ autoQuoteTableNames: true, autoQuoteFieldNames: true })
+      .from(this.tableName)
+      .where(`${this.pkColumnName} = ?`, uid)
```

Comment:
So, pkColumnName isn't really defined for this class because the Scores table actually uses two columns as its primary key. For this, probably just change the references to ${this.pkColumnName} to be 'project_id' and 'judge'.